packages/mobile/capacitor/capacitorApi.ts

```diff
@@ -417,17 +424,17 @@ export const CapacitorApi = {
      * @returns
      */
     saveRecoveryKit: async (recoverKitData) => {
-        // const os: string = Capacitor.getPlatform()
-        // const { selected } = await SecureFilesystemAccess.showPicker({
-        //     type: 'folder',
-        //     defaultPath: '',
-        // })
-        // if (os === 'ios') {
-        //     void (await SecureFilesystemAccess.allowAccess())
-        // }
-        // if (os === 'ios') {
-        //     void SecureFilesystemAccess.revokeAccess()
-        // }
+        const os: string = Capacitor.getPlatform()
+        const { selected } = await SecureFilesystemAccess.showPicker({
+            type: 'folder',
+            defaultPath: '',
+        })
+        if (os === 'ios') {
```

Comment:
I guess we do not need both if statements, right? Do we even need the function "saveRecoveryKit" for mobile?